halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/account/NewRelicAddCanaryAccountCommand.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Google, Inc.
```

Comment:
oh and also, these headers probably should be `2019 NewRelic, Inc.` or your company's entity type. (there's a couple in the PR)

Reply:
Thanks for catching that @kevinawoo - made those changes across all the files